.github/workflows/ci.yml

```diff
@@ -39,18 +39,21 @@ jobs:
           cache-dependency-path: 'pnpm-lock.yaml'
 
       - name: Install Dependencies
-        run: pnpm install
+        run: pnpm install # note this runs pre-build
 
-      - name: Store Prune
-        run: pnpm store prune
+      - name: Prebuild Bin Executables
+        run: pnpm run pre-build
```

Comment:
This is technically happening during install due to the `prepare` script but this is just a sanity run